src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -21,6 +21,6 @@
                 :align-items      :center
                 :justify-content  :center
                 :border-radius    (/ container-size 2)
-                :background-color (colors/custom-color-by-theme color 50 60)}
+                :background-color (colors/custom-hex-color color 50 60)}
```

Comment:
could you please elaborate?

Reply:
`custom-color-by-theme` is not working

Reply:
Please, also check quo2 preview while modifying quo2 components. This change is breaking the component

Reply:
I think we need to give the preview component a list of hex values?

Reply:
We are going to get rid of old colors and designs, So I would recommend we can check if provided color is a keyword or not. And if not fallback to `:primary`
(Not sure, but we probably not going to support group theme customization for MVP? cc: @pedro-et).

Also using stored value will not be helpful, because those colors might not work with the current app background colors, and changing the group avatar to cover this case of hard-coded color instead of dynamic, might create an unwanted chain reaction (changes in other places)